packages/contracts/src/core/dao/legacy/IDAO_v1_0_1.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Can't these legacy files be moved to a folder that doesn't pollute the main codebase?
If the goal is to test the upgradeability, I would create a specific spot within that folder and keep them there

Reply:
Moved in dbe21345ea8987092a6e68da49282b5a6f7bdd7c